local-tests/tests/testSignTransactionWithSolanaEncryptedKey.ts

```diff
@@ -0,0 +1,131 @@
+import { log } from '@lit-protocol/misc';
+import { TinnyEnvironment } from 'local-tests/setup/tinny-environment';
+import { signTransactionWithSolanaEncryptedKeyLitActionIpfsCid } from '@lit-protocol/wrapped-keys';
+import {
+  AccessControlConditions,
+  ILitNodeClient,
+  LitAbility,
+} from '@lit-protocol/types';
+import { encryptString } from '@lit-protocol/lit-node-client-nodejs';
+import {
+  Connection,
+  Keypair,
+  LAMPORTS_PER_SOL,
+  PublicKey,
+  SystemProgram,
+  Transaction,
+  clusterApiUrl,
+} from '@solana/web3.js';
+import {
+  LitAccessControlConditionResource,
+  LitActionResource,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from '@lit-protocol/auth-helpers';
+
+/**
+ * Test Commands:
+ * ✅ NETWORK=cayenne yarn test:local --filter=testSignTransactionWithSolanaEncryptedKey
+ * ✅ NETWORK=manzano yarn test:local --filter=testSignTransactionWithSolanaEncryptedKey
+ * ❔ NETWORK=localchain yarn test:local --filter=testSignTransactionWithSolanaEncryptedKey
+ */
+export const testSignTransactionWithSolanaEncryptedKey = async (
+  devEnv: TinnyEnvironment
+) => {
+  const alice = await devEnv.createRandomPerson();
+  const solanaKeypair = Keypair.generate();
+  const decryptionAccessControlConditions: AccessControlConditions = [
+    {
+      contractAddress: '',
+      standardContractType: '',
+      chain: 'ethereum',
+      method: '',
+      parameters: [':userAddress', 'latest'],
+      returnValueTest: {
+        comparator: '=',
+        value: await alice.wallet.getAddress(),
+      },
+    },
+  ];
+
+  const { ciphertext, dataToEncryptHash } = await encryptString(
+    {
+      accessControlConditions: decryptionAccessControlConditions,
+      dataToEncrypt: Buffer.from(solanaKeypair.secretKey).toString('hex'),
+    },
+    devEnv.litNodeClient as unknown as ILitNodeClient
+  );
+
+  const sessionSigs = await devEnv.litNodeClient.getSessionSigs({
+    chain: 'ethereum',
+    resourceAbilityRequests: [
+      {
+        resource: new LitActionResource('*'),
+        ability: LitAbility.LitActionExecution,
+      },
+      {
+        resource: new LitAccessControlConditionResource('*'),
+        ability: LitAbility.AccessControlConditionDecryption,
+      },
+    ],
+    authNeededCallback: async ({ uri, resourceAbilityRequests }) => {
+      const toSign = await createSiweMessageWithRecaps({
+        uri: uri!,
+        expiration: new Date(Date.now() + 1000 * 60 * 10).toISOString(),
+        resources: resourceAbilityRequests!,
+        walletAddress: await alice.wallet.getAddress(),
+        nonce: await devEnv.litNodeClient!.getLatestBlockhash(),
+        litNodeClient: devEnv.litNodeClient,
+      });
+      return await generateAuthSig({
+        signer: alice.wallet,
+        toSign,
+      });
+    },
+  });
```

Comment:
Same as in the above file

Reply:
Addressed via [this commit](https://github.com/LIT-Protocol/js-sdk/pull/504/commits/c538a1e8d6011a0929e13e2d4c82ae7241d45cf1)

Reply:
As commented above, it should be `getPkpSessionSigs` which gives you the PKP sessionSigs not EoaSessionSig as we only allow PKP sessionSigs to use Wrapped Key